packages/gatsby/src/utils/webpack.config.js

```diff
@@ -259,6 +259,20 @@ module.exports = async (
       rules.media(),
       rules.miscAssets(),
     ]
+    if (
+      store.getState().config._experimentalThemes &&
```

Comment:
We should make this the same as the [component shadowing check](https://github.com/gatsbyjs/gatsby/blob/bc7b0956efa29fcd5c74dabda8a6641fc2cccb19/packages/gatsby/src/internal-plugins/webpack-theme-component-shadowing/gatsby-node.js#L9).

```suggestion
      store.getState().themes.themes
```